gradle.properties

```diff
@@ -94,6 +94,8 @@ activationVersion=1.2.2
 
 annotationsVersion=15.0
 
+apacheAntVersion=1.10.13
```

Comment:
I see we've become a little inconsistent with naming convention, but usually this would just be the name the artifact (`ant`) followed by `version` (so `antVersion`).

Reply:
Can we capture our desired convention in a comment in the file itself? It won't guarantee compliance, of course, but it's more likely than leaving it unstated.

Reply:
Updated the variable, add a comment at the top of the section to include the naming convention.